aldor/aldor/src/assert.h0

```diff
@@ -18,8 +18,7 @@
 
 #ifndef DO_ASSERT_IS_DECLARED
 #define DO_ASSERT_IS_DECLARED
-void _abort_if_fatal_assert(void) chk_noreturn;
-void _do_assert(char *str, char *file, int line) chk_noreturn;
+void _do_assert(char *str, char *file, int line);
```

Comment:
This is still noreturn, right?

Reply:
no - do_assert can return in some situations (annoyingly).  Maybe it would be better to remove that feature.